### ISSUE TITLE
Title of F42 indicates it applies to SC 4.1.2 - but 4.1.2 is not listed in the technique

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#1263]
4.1.2 is not mentioned in the "This technique relates to:" section
https://www.w3.org/WAI/WCAG21/Techniques/failures/F42.html

4.1.2 is mentioned at bottom in check section.
